test/functorch/test_aotdispatch.py

```diff
@@ -1015,7 +1015,6 @@ def assert_compiler(gm: torch.fx.GraphModule, _):
     xfail('cumprod', ''),  # aten.cumprod.default - couldn't find symbolic meta function/decomposition
     xfail('cumsum', ''),  # aten.cumsum.default - couldn't find symbolic meta function/decomposition
     xfail('cumulative_trapezoid', ''),  # Cannot call sizes() on tensor with symbolic sizes/strides
-    xfail('deg2rad', ''),  # aten.deg2rad.default - couldn't find symbolic meta function/decomposition
     xfail('diff', ''),  # aten.zeros_like.default - couldn't find symbolic meta function/decomposition
```

Comment:
shouldn't this start passing after adding the zeros_like meta impl above?

Reply:
Probably hit another error